docker/docker-compose.yml

```
@@ -5,18 +5,19 @@ services:
       context: ..
       dockerfile: docker/Dockerfile
       args:
-        - TAG=humble-0.21
+        - TAG=foxy-0.21
     environment:
       DISPLAY: $DISPLAY
       XAUTHORITY: $XAUTHORITY
       NVIDIA_DRIVER_CAPABILITIES: all
       ROS_LOG_DIR: /tmp
     container_name: snp
-    image: ghcr.io/ros-industrial-consortium/scan_n_plan_workshop:humble-latest
+    image: ghcr.io/ros-industrial-consortium/scan_n_plan_workshop:foxy-latest
     stdin_open: true
     tty: true
     network_mode: host
-    user: ${CURRENT_UID}  # CURRENT_UID=$(id -u):$(id -g)
+    privileged: true
+    user: root #${CURRENT_UID}  # CURRENT_UID=$(id -u):$(id -g)
     volumes:
       - /tmp/.X11-unix:/tmp/.X11-unix
       - /etc/hosts:/etc/hosts
```

```suggestion
      - /tmp/.X11-unix:/tmp/.X11-unix
      - /etc/hosts:/etc/hosts
      - /etc/passwd:/etc/passwd:ro
      - /etc/group:/etc/group:ro
```
Let's add this too so the container recognizes the user name and password